targets.json

```diff
@@ -0,0 +1,446 @@
+[
```

Comment:
should this maybe be toml?

Reply:
something like that yeah, I'm not sure if I have a preference

Reply:
I think TOML's much more compact and legible in this case, so I quite like it. Since we're not doing nested keys, this is a perfect use-case for TOML.

Reply:
switched to toml, while documenting the fields I noticed some improvements we can do also, so that's nice